inlong-dataproxy/conf/common.properties

```diff
@@ -23,14 +23,15 @@ manager.hosts=127.0.0.1:8083
 manager.auth.secretId=
 manager.auth.secretKey=
 proxy.report.ip=127.0.0.1
-proxy.report.port=46801
+proxy.report.tcp.port=46801
+proxy.report.http.port=46802
+proxy.report.tcp.protocol=tcp
 # proxy cluster name
 proxy.cluster.name=default_dataproxy
 proxy.cluster.tag=default_cluster
 proxy.cluster.inCharges=admin
 # check interval of local config (millisecond)
 configCheckInterval=10000
-
```

Comment:
Please keep this blank line, it is more readable.

Reply:
has moved another PR